src/mgos_prometheus_metrics.c

```diff
@@ -165,11 +166,25 @@ static void metrics_handle(struct mg_connection *nc, int ev, void *ev_data,
   (void) user_data;
 }
 
+static void metric_rpc_cb(struct mg_rpc_request_info *ri, void *cb_arg,
+                   struct mg_rpc_frame_info *fi, struct mg_str args) {
+  
+  // populate metric data
+  mg_rpc_send_responsef(ri, "");
```

Comment:
@rojer @pimvanpelt looks like this feature is non-trivial, for this feature there would require a lot of refactoring as `mgos_prometheus_metrics_send_chunks` uses `mg_printf` to write to connection `nc`.

For RPC `mg_rpc_send_responsef` requires `char[]`.

Is there a workaround to do this without refactoring?